jujugui/static/gui/src/app/components/account/payment-methods/card/card.js

```diff
@@ -44,6 +55,47 @@ YUI.add('account-payment-method-card', function() {
       this.setState({cardFlipped: !this.state.cardFlipped});
     },
 
+    /**
+      Remove the payment method.
+
+      @method _removePaymentMethod
+    */
+    _removePaymentMethod: function() {
+      const xhr = this.props.removePaymentMethod(
+        this.props.username, this.props.card.name, error => {
+          if (error) {
+            const message = 'Unable to remove the payment method';
+            this.props.addNotification({
+              title: message,
+              message: `${message}: ${error}`,
+              level: 'error'
+            });
+            console.error(message, error);
+            return;
+          }
+          this.props.onPaymentMethodRemoved();
+        });
+      this.xhrs.push(xhr);
+    },
+
+    /**
+      Generate the card actions.
+
+      @method _generateActions
+    */
+    _generateActions: function() {
+      if (!this.props.removePaymentMethod) {
```

Comment:
This component gets reused in the deployment flow, in that case we don't allow the payment method to be removed so we don't need those methods (but maybe we should, or at least link to the account page).